tools/build-release-branch.sh

```diff
@@ -238,6 +238,13 @@ yarn cache clean
 COMPOSER_MIRROR_PATH_REPOS=1 yarn run build-production
 echo "Done"
 
+# Remove composer dependencies listing.
+# Workaround for https://github.com/Automattic/jetpack/issues/13497
+# TODO: Keep dependencies away from built branches and fetch them before publishing on WP-org plugins repository.
+awk '!/automattic\/jetpack-/' composer.json > temp && mv temp composer.json
+sed 's/openssl": "\*",/openssl": "\*"/' composer.json > temp && mv temp composer.json
+rm -f composer.lock
```

Comment:
That should not be necessary. I've just pushed #14905 to ensure it never makes it to the built branch.

Reply:
Ok!

Reply:
@jeherve The `composer.lock` seems to still be included in built branches: https://github.com/Automattic/jetpack/blob/branch-8.4-built/composer.lock

Are you sure we should not add this line anyway?
Thx!

Reply:
This was because of a hiccup when building the built branch the first time around. It will not happen in the future.